data/templates/ipsec/ipsec.conf.tmpl

```diff
@@ -1,3 +1,121 @@
+# Created by VyOS - manual changes will be overwritten
+
+{% import 'ipsec/ike-esp.tmpl' as ike_esp %}
+
+config setup
+    charondebug = "{{ charondebug }}"
+    uniqueids = {{ "no" if disable_uniqreqids is defined else "yes" }}
+
+{%  if site_to_site is defined and "peer" in site_to_site -%}
+{%      for peer, peer_conf in site_to_site.peer.items() %}
+{%          set peer_index = loop.index %}
+{%          set peer_ike = ike_group[peer_conf.ike_group] %}
+{%          set peer_esp = esp_group[peer_conf.default_esp_group] if 'default_esp_group' in peer_conf else None %}
+conn site{{peer_index}} # peer:{{ peer }}
```

Comment:
Is it possible to use the old format for the connection name?
```
vyos@r4-1.3:~$ show vpn ipsec sa
Connection    State    Uptime    Bytes In/Out    Packets In/Out    Remote address    Remote ID    Proposal
------------  -------  --------  --------------  ----------------  ----------------  -----------  ----------------------
site1t0       up       4m56s     0B/0B           0/0               100.64.0.2        N/A          AES_GCM_16_256
site1t1       up       4m56s     0B/0B           0/0               100.64.0.2        N/A          AES_GCM_16_256/ECP_256

```
Old format:
```
vyos@r6-roll:~$ show vpn ipsec sa
Connection                State    Uptime    Bytes In/Out    Packets In/Out    Remote address    Remote ID    Proposal
------------------------  -------  --------  --------------  ----------------  ----------------  -----------  ----------------------
peer-100.64.0.1-tunnel-0  up       3m57s     0B/0B           0/0               100.64.0.1        N/A          AES_GCM_16_256
peer-100.64.0.1-tunnel-1  up       3m57s     0B/0B           0/0               100.64.0.1        N/A          AES_GCM_16_256/ECP_256
vyos@r6-roll:~$ 
```

Some customers use their own scripts for checking ipsec status match old format style.